app/src/main/java/co/eventbox/event/respository/Repository.kt

```diff
@@ -21,8 +22,9 @@ abstract class Repository<D : Operation.Data, V : Operation.Variables, O : Opera
     ResultFetcher<O, D> {
 
 
-    private val token = "56fe1591-c5c1-4113-aba3-73fca1f5aacd" // English Data Token
-    private val okHttpProvider = OkHttpClientProvider.provide(token)
+    private val token1 = "56fe1591-c5c1-4113-aba3-73fca1f5aacd" // English Data Token
```

Comment:
Now we hold the token in two places as a field
Repository and Constants

Reply:
All tokens must be read from the same file and field.

Please remove token1 and use PERSIAN_TOKEN or ENGLISH_TOKEN in Constants object